ONSdigital/response-operations-ui
# [TBL] Call party service directly to change contact details

# Motivation and Context

Removes the call to backstage for changing a party's contact details.

# What has changed



Changes the `update_contact_details` method (called by `/reporting-units/<ru_ref>/edit-contact-details/<respondent_id>`) in party_controller.py to hit the party service directly.

# How to test?



Run the unit tests, make sure they pass.
Tail the logs for backstage, party service. Hit the `/reporting-units/<ru_ref>/edit-contact-details/<respondent_id>` endpoint using `curl` or Postman. Make sure backstage doesn't get the call, but party service does.
